lib/manager/components/version/FeedVersionTabs.js

```diff
@@ -69,10 +71,12 @@ export default class FeedVersionTabs extends Component<Props> {
   render () {
     const {
       feedVersionIndex,
+      comparedVersionIndex,
```

Comment:
This seems like unnecessary passing of props which could be derived from the comparedVersion. There's probably no precalculation time-savings from doing it this way.

Reply:
I removed this redundant prop in https://github.com/ibi-group/datatools-ui/pull/528/commits/03695eda1153b68d93751b1f0d27738edc2b41b1.

Reply:
There is also a `feedVersionIndex` prop that is not germaine to this PR. Let me know if I need to address it as well.